app/utils/url_utils.rb

```diff
@@ -43,4 +43,13 @@ def extract_locale_from_url(url)
   def strip_port_from_host(host)
     host.split(":").first
   end
+
+  # Naive join method, which can be used to normalize multiple slashes
+  #
+  # Usage: URLUtils.join("foo", "bar/", "baz") => "foo/bar/baz"
+  def join(*parts)
```

Comment:
I'd expect that `join("foo", "/bar") == "foo/bar"`. At least the method name suggests it's all about joining components and also Ruby's own `File.join` works similarly. I'd think of it like this: User wants a path relative to the arbitrary root of their location (the `/bar` part) and `foo` is the system wide prefix for that.